apps/website/contents/resume.md

```diff
@@ -28,9 +28,9 @@ Having read tons of software engineering resumes as a FAANG interviewer, even so
 Thankfully, there are specific steps and requirements you can fulfill to write a good software engineer resume. From your resume structure, content, to free tools you can use to test your resume, I have collated a very concise summary of the best practices to prepare your resume for a FAANG software engineering job application:
 
 1. Set up an ATS-friendly resume template
-1. Fill up your template with well-framed content in a meaningful order
-1. Optimize your resume with prioritization and keywords
-1. Test out resume using free tools
+2. Fill up your template with well-framed content in a meaningful order
```

Comment:
Please revert the changes to the sections to the PR. Using `1.` for all the bullet points is intended, so that we can insert new bullet points anywhere and the rendered numbers are still in order.

Reply:
Fixed ✅